grafana-plugin/src/containers/MobileAppConnection/MobileAppConnection.tsx

```diff
@@ -23,7 +23,8 @@ import QRCode from './parts/QRCode/QRCode';
 const cx = cn.bind(styles);
 
 type Props = {
-  userPk: User['pk'];
+  userPk?: User['pk'];
+  // store: typeof rootStore;
```

Comment:
is `store` not needed here anymore?

Reply:
hmm was it ever needed?

Reply:
it's just the comment, needs to be removed